Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.6
+
+RUN apt-get update && \
+    apt-get install -y && \
+    pip3 install uwsgi
+
+COPY . /code
+WORKDIR /code
+RUN pip3 install -r /code/requirements-dev.txt
+
+ENV DJANGO_ENV=prod
```

Comment:
If your env variables are dev settings, please use DJANGO_ENV=dev.

Reply:
This environment variable is actually not being used. Just put in there in case future use case. I will remove it.